doc/modules/model_evaluation.rst

```diff
@@ -1193,6 +1194,45 @@ if the predicted outputs have been binarized.
     for an example of using ROC to
     model species distribution.
 
+.. _det_curve:
+
+Detection error tradeoff (DET)
+---------------------------------------
+
+The function :func:`detection_error_tradeoff_curve` computes the
+`detection error tradeoff curve, or DET curve <https://en.wikipedia.org/wiki/Detection_error_tradeoff>`_.
+Quoting Wikipedia :
+
+  "A detection error tradeoff (DET) graph is a graphical plot of error rates for binary classification systems, plotting false reject rate vs. false accept rate. The x- and y-axes are scaled non-linearly by their standard normal deviates (or just by logarithmic transformation), yielding tradeoff curves that are more linear than ROC curves, and use most of the image area to highlight the differences of importance in the critical operating region."
```

Comment:
please limit line lengths to 80 characters. Blockquotes can be indented.

Could you add discussion of how to use this for model selection / evaluation  / critique?